django_colortag/models.py

```diff
@@ -83,4 +83,7 @@ def save(self, *args, **kwargs):
 
         self.slug = slug_candidate
 
-        return super().save(*args, **kwargs)
+        if update_fields is not None:
+            update_fields = tuple(set(update_fields) | {"slug"})
```

Comment:
Why is this necessary? It makes sense since the `slug` field is modified in the `save()` method, but why did you add it now?